README.md

```diff
@@ -104,24 +104,32 @@ details.
 helm repo update
 # See current Chart & Traefik version
 helm search repo traefik/traefik
-# Update CRDs (Traefik Proxy v3 CRDs)
+# Update CRDs
 kubectl apply --server-side --force-conflicts -k https://github.com/traefik/traefik-helm-chart/traefik/crds/
```

Comment:
Although this is probably fine, I usually prefer to use the version that comes with my downloaded chart to ensure versions match with what I want (rather than just using the latest) or use some tagged version from upstream.

However, after looking into this, I think the following is probably a neat way to achieve this (only tested this with `kubectl diff` rather than applying, but that seems to look ok):
`helm show crds traefik/traefik | kubectl apply --server-side --force-conflicts -f -`

Reply:
\o That's a great idea ! We may then even remove this kustomization.yaml !

Reply:
I've added it with cf41d2f20fb7c182a45c825d37645bd3788830d4.
I'll double check it tomorrow morning